Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -517,13 +517,15 @@ def start_download_from_uri(self, uri):
 
         return None
 
+    @blocking_call_on_reactor_thread
```

Comment:
Use @deferred(timeout) instead, it will push the call to the reactor thread and wait for the deferred to resolve too